### DISCUSSION TITLE
Having a separate board with display only for status check?

### DISCUSSION THREAD

**Discussion by greymda** [hoylabs/OpenDTU-OnBattery#965] [General]
So most probably, when all bits arrive, my build will be living in the basement (battery, mppt, inverter and the ESP with ODoB) but i really got used to glance the display with the stats.
Question: can we setup a second build so it only displays the data/statuses without any commands and so on? Like have a _master_ board and a _slave_ board?
Cheers

**Comment by genmad** [comment]
Like this build here: https://github.com/helgeerbe/OpenDTU-OnBattery/discussions/619#discussioncomment-9337486

**Reply by greymda** [reply]
yep, something similar.

**Comment by spcqike** [comment]
You could use tasmota with a display 

listen to your openDTUs MQTT messages and display the values accordingly 

https://tasmota.github.io/docs/Displays/#rule-examples

**Comment by greymda** [comment]
getting there, bit by bit 
![IMG_5811](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/9991596/48566bd8-5520-4e92-810f-52ec0631a5b4)

**Comment by greymda** [comment]
i think it’s okay
![IMG_5812](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/9991596/084aa420-45b8-4fdb-95c4-87a94da09957)

**Reply by spcqike** [reply]
glad you found a working solution, especially with components you allready had.

i guess i would have turned the display by 90 degree for something like this

```  
   ┌──────────────┬───────────────┐  
   │ PV           │ Grid          │  
   │              │               │  
   │              │               │  
   ├──────────────┴───────────────┤  
   │   Today                      │  
   │                              │  
   │                              │  
   └──────────────────────────────┘  
```

**Reply by greymda** [reply]
have to figure out how. it’s quite nasty to use the DisplayText, at least for me.
